modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,117 @@ If you have installed the {gitops-title} Operator in the default namespace, run
 $ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
 ----
 
-[id="breaking-change-1-15-0_{context}"]
-== Breaking change
+[id="new-features-1-15-0_{context}"]
+== New features
+
+* With this update, {gitops-title} supports the addition of extra volumes and volume mounts to the `ApplicationSet` Controller components. You can configure these additional resources by specifying the `spec.applicationSet.volumes` and `spec.applicationSet.volumeMounts` fields in the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-5739[GITOPS-5739]
++
+.Example
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-example
+spec:
+  applicationSet:
+    volumes:
+      - name: <volume-name> #<1>
+        emptyDir: {} #<2>
+    volumeMounts:
+      - name: <volume-name> #<3>
+        mountPath: <mount-path> #<4>
+----
+<1> Specifies the name of the `volume` for the Argo CD `ApplicationSet` Controller.
+<2> Specifies the `volume` type. It can be any Kubernetes supported volume type, such as `emptyDir`, `hostPath`, `configMap`, and so on.
+<3> Specifies the name of the volume to be mounted, which must match the name of the `volume` defined in volumes.
+<4> Specifies the path inside the container where the volume will be mounted.
+
+* With this update, users can add custom labels and annotations to the server, repo, applicationset-controller, and application-controller pods. This enhancement improves observability for external systems that depend on resource labels or annotations for monitoring purposes. The following fields are introduced in the Argo CD CR to support this functionality:
+
+** `.spec.server.labels`
+** `.spec.server.annotations`
+** `.spec.repo.labels`
+** `.spec.repo.annotations`
+** `.spec.controller.labels`
+** `.spec.controller.annotations`
+** `.spec. applicationSet.labels`
+** `.spec. applicationSet.annotations` 
+
+For more information, see link:https://argocd-operator.readthedocs.io/en/latest/usage/custom_labels_annotations/[Custom Annotations and Labels]. link:https://issues.redhat.com/browse/GITOPS-5674[GITOPS-5674]
+
+* Before this update, users could not configure some replicas for the Argo Rollouts controller by using the `RolloutManager` custom resource (CR). With this update, high availability (HA) support is introduced by adding a new `.spec.ha` field to the `RolloutManager` CR.
```

Comment:
nitpick: If we are adding RolloutManager in backticks, probably we do the same for ArgoCD CR?

Reply:
No, I would rather keep ArgoCD CR as is.